.readthedocs.yaml

```diff
@@ -0,0 +1,26 @@
+version: 2
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "mambaforge-4.10"
+
+conda:
+  environment: docs/rtd_environment.yaml
```

Comment:
I don't think you need to use conda - if this is because you need graphviz I *think* it is installed by default (I think worth checking) but if not you can also just specify it like this:

https://github.com/readthedocs/readthedocs.org/issues/8800#issuecomment-1007261093

We use graphviz in astropy core but I think we just rely on it being there by default.